src/fabric/src/fabric2_fdb.erl

```diff
@@ -307,6 +312,54 @@ exists(#{name := DbName} = Db) when is_binary(DbName) ->
     end.
 
 
+create_iter(#{tx := undefined} = Db) ->
+    try
+        Db1 = refresh(Db),
+
+        Reopen = maps:get(reopen, Db1, false),
+        Db2 = maps:remove(reopen, Db1),
+
+        Fdb = get_db_handle(),
+        Tx = erlfdb:reate_read_only_transaction(Fdb),
```

Comment:
I think that should be `create_read_only_transaction

Reply:
Good catch